macros/MakeSimpleQuicklinks.ejs

```diff
@@ -13,24 +13,24 @@
 // {{MakeSimpleQuicklinks("/en-US/docs/foo", "http://somesite.com|Awesome Site",
 //     "https://othersite.com|Another site|Tooltip for another site")}}
 
-var quicklinks = "<section id='Quick_Links'><ol>";
-var count = arguments.length;
+let quicklinks = "<section id='Quick_Links'><ol>";
+let count = arguments.length;
 
-for (var i=0; i<count; i++) {
-    var item = arguments[i];
-    var path = "";
-    var title = "";
-    var text = "";
+for (let i=0; i<count; i++) {
```

Comment:
Nit: Not introduced here but, the preferred code style is:

```suggestion
for (let i = 0; i < count; i++) {
```

Reply:
Not going to touch that in this PR.